deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/drawer/StakeMeedsDrawer.vue

```diff
@@ -19,7 +19,7 @@
 <template>
   <deeds-drawer ref="drawer">
     <template #title>
-      <h4>{{ title }}</h4>
+      <h4 class="title font-weight-bold">{{ title }}</h4>
```

Comment:
It's may be better to define generic CSS for `h4` to use a different font-size in all dapp.

Reply:
done!